src/BUILD

```diff
@@ -38,8 +38,8 @@ filegroup(
         "//conditions:default": md5_cmd % "md5sum",
     }),
 ) for suffix, embedded_tools_target in {
-    "": [":embedded_tools"],
-    "_with_jdk": [":embedded_tools_with_jdk"],
+    "": [":embedded_tools_nojdk"],
```

Comment:
did you mean "embedded_tools" here, and "embedded_tools_nojdk" below?

Reply:
Yep, thanks. Done.